lib/compress/zstdmt_compress.c

```diff
@@ -447,8 +449,6 @@ ZSTDMT_CCtx* ZSTDMT_createCCtx_advanced(unsigned nbThreads, ZSTD_customMem cMem)
     ZSTDMT_initializeCCtxParameters(&mtctx->params, nbThreads);
     mtctx->cMem = cMem;
     mtctx->allJobsCompleted = 1;
-    mtctx->sectionSize = 0;
```

Comment:
This seems like an unrelated change, why do we need it?

Reply:
This was to fix the merging with the parameters code. The mtctx->sectionSize field no longer exists and was moved to `CCtx_params`.